doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -58,26 +61,30 @@ GNU ARM Embedded
       - Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``gnuarmemb``.
       - Set :envvar:`GNUARMEMB_TOOLCHAIN_PATH` to the brew installation directory (something like ``/usr/local``)
 
-Intel oneApi Toolkits
+Intel oneAPI Toolkit
 *********************
 
 #. Download `Intel oneAPI Base Toolkit
    <https://software.intel.com/content/www/us/en/develop/tools/oneapi/all-toolkits.html>`_
 
-#. Assuming the toolkit is installed in ``/opt/intel/oneApi``, set environment
+#. Assuming the toolkit is installed in ``/opt/intel/oneapi``, set environment
    using::
 
+        # Linux, macOS:
         export ONEAPI_TOOLCHAIN_PATH=/opt/intel/oneapi
         source $ONEAPI_TOOLCHAIN_PATH/compiler/latest/env/vars.sh
 
+        # Windows:
+        > set ONEAPI_TOOLCHAIN_PATH=C:\Users\Intel\oneAPI
+
    To setup the complete oneApi environment, use::
 
         source  /opt/intel/oneapi/setvars.sh
 
    The above will also change the python environment to the one used by the
    toolchain and might conflict with what Zephyr uses.
 
-#. Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``oneApi``.
+#. Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``oneapi``.
```

Comment:
this will break the system.
On windows, things will work, as the filesystem in case-insensitive, but other OS'es has case sensitive file systems.

The `ZEPHYR_TOOLCHAIN_VARIANT` is used here:
https://github.com/zephyrproject-rtos/zephyr/blob/04e70a482de5c171d7e48eca2101f87735d26b4f/cmake/target_toolchain.cmake#L36
(and maybe more places).
and the oneApi folder is named `oneApi` not `oneapi`:
https://github.com/zephyrproject-rtos/zephyr/tree/main/cmake/toolchain/oneApi

So before changing casing on that environment variable from `oneApi` --> `oneapi`, please ensure that things are still working on all platforms.

Reply:
Thank you for the review, Torsten. I removed that part, left as it was before my changes.